src/openrct2/scripting/bindings/world/ScMap.cpp

```diff
@@ -167,6 +167,67 @@ namespace OpenRCT2::Scripting
         return result;
     }
 
+    std::vector<DukValue> OpenRCT2::Scripting::ScMap::getAllEntitiesOnTile(
+        const std::string& type, const DukValue& tilePos) const
+    {
+        // Get the tile position
+        const auto pos = FromDuk<CoordsXY>(tilePos);
```

Comment:
I think it would be better to take TileCoords instead of CoordsXY, so each tile increments X and Y by only 1 instead of 32. To get all entities on tile `{x: 70, y: 75}` for example, you now have to supply `{x: 70 * 32, y: 75 * 32}`.

Reply:
https://github.com/Broxzier/OpenRCT2/commit/6ee6ae3906ac77037d67a778804d2c78b517ba41

Reply:
I agree, tiles should use tile coordinates.

Reply:
The use of big coords and tile coords seems to be rather mixed in the API. I've seen a couple plugins use game-actions where they multiply every value with 32 to iterate over some tiles, and the getTile function on the map takes two arguments for x and y where they are tile coords instead.

Reply:
Is this solved (and thus PR mergeable?)

Reply:
Well it still takes big coords instead of tile coords. I tried changing it so that the pos was multiplied by 32, but my build environment get a little messed up after the new save format update was released. Is there anything special I'm supposed to do to update it? Running from the game launcher works fine but Visual Studio logs a thousand "Object not found." errors and I *think* it tries to load the older format

Reply:
you need to do a full rebuild (from the msbuild window) to make it download the updated objects.

Reply:
I think I tried that, but I'll try again. If that dosen't work I'll just delete the local repository and clone it again

Reply:
It seems that the use of TileCoords and Coords are a bit mixed in the API. This is better left for another issue.